infrastructure/comms/shared_struct/shared_struct_publisher.inl

```diff
@@ -0,0 +1,43 @@
+
+namespace jet {
+
+SharedStructPublisher::SharedStructPublisher(const std::string& object_name) : Publisher(object_name) {
+  int o_flags = O_RDWR | O_CREAT;
+  off_t length = 921650 + 1024;
+  int shmem_file_descriptor_ = shm_open(object_name.c_str(), o_flags, 0644 );
+  if (shmem_file_descriptor_ < 0) {
+    std::cerr << "Error opening shared memory region: " <<  errno << std::endl;
+    throw std::runtime_error("Could not open shared memory region.");
```

Comment:
Consider only throwing the error (and not printing)

Reply:
:+1: